main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/joho/godotenv"
+	"main.go/database"
+	"main.go/parsedate"
+	"main.go/tasks"
+	_ "modernc.org/sqlite"
+)
+
+const (
+	defPort   = "7540"
+	webDir    = "./web"
+	defDBFile = "./scheduler.db"
+)
+
+func main() {
+	var db *sql.DB
+	var err error
+	db, err = database.InitDatabase()
+	if err != nil {
+		log.Fatalf("Bad DB: %v\n", err)
+	}
+	defer db.Close()
+
+	envik := godotenv.Load()
```

Comment:
Не очевидно, что это ошибка
```suggestion
	envikOshibka := godotenv.Load()
```